### PR TITLE
Change to unary mixer API

### PR DESCRIPTION
**Pull request by kyessenov**
Works but the HEADs of api/proxy/mixer don't work together.
cc @geeknoid @qiwzhang @ZackButcher

### CODE REVIEW

**Comment by qiwzhang** [comment]
"mixer_server" field should be removed from filter config.

File: proxy/envoy/resources.go [hunk 1]
```diff
@@ -150,9 +159,6 @@ type Header struct {
 
 // FilterMixerConfig definition
 type FilterMixerConfig struct {
-	// MixerServer specifies the address of the mixer server (e.g. "mixer:9090")
```
Comment (kyessenov) [inline proxy/envoy/resources.go]: @qiwzhang it's removed

File: proxy/envoy/testdata/envoy-v1.json.golden [hunk 2]
```diff
@@ -439,6 +439,24 @@
         ]
       },
       {
+        "name": "mixer_server",
```
Comment (mandarjog) [inline proxy/envoy/testdata/envoy-v1.json.golden]: Is mixer_server an upstream / cluster now ?
How do statically defined clusters interact with CDS ?

Comment (kyessenov) [inline proxy/envoy/testdata/envoy-v1.json.golden]: there is no CDS for mixer, not an issue.

**Comment by kyessenov** [comment]
fixes #820

**Pull request merged**